src/beamlime/dashboard/monitors.py

```diff
@@ -70,7 +83,8 @@ def create_sidebar_content(self) -> pn.viewable.Viewable:
             pn.pane.Markdown("## Controls"),
             self._view_toggle_group,
             StartTimeWidget(self._reset_controller).panel,
-            TOAEdgesWidget(self._toa_controller).panel,
+            pn.pane.Markdown("## Workflows"),
+            self._reduction_widget.widget,
```

Comment:
TL;DR The monitor dashboard now uses the same dialog-based mechanism as the reduction dashboard. The specific TOA-edge controls are gone.